DrupalExtended/ruleset.xml

```diff
@@ -0,0 +1,164 @@
+<?xml version="1.0"?>
+<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="Drupal Extended" xsi:noNamespaceSchemaLocation="../vendor/squizlabs/php_codesniffer/phpcs.xsd">
+  <description>An extension of Drupal coding standards.</description>
+
+  <rule ref="vendor/drupal/coder/coder_sniffer/Drupal">
+    <!-- Exclude in favor of native typehints. See: https://www.drupal.org/project/coding_standards/issues/3238192 -->
+    <exclude name="Drupal.Commenting.VariableComment.MissingVar"/>
+    <!-- Conflicts with "SlevomatCodingStandard.TypeHints.DeclareStrictTypes". -->
+    <exclude name="Drupal.WhiteSpace.OpenTagNewline.BlankLine"/>
+    <!-- Conflicts with "<?php declare(strict_types = 1)." -->
+    <exclude name="Drupal.Commenting.FileComment.Missing"/>
+  </rule>
+
+  <rule ref="vendor/drupal/coder/coder_sniffer/DrupalPractice"/>
+
+  <rule ref="vendor/slevomat/coding-standard/SlevomatCodingStandard">
+    <!-- Excluded rules based on what enabled in 1.x branch. -->
+    <exclude name="SlevomatCodingStandard.Arrays.DisallowImplicitArrayCreation"/>
+    <exclude name="SlevomatCodingStandard.Arrays.MultiLineArrayEndBracketPlacement"/>
+    <exclude name="SlevomatCodingStandard.Arrays.SingleLineArrayWhitespace"/>
+    <!-- Same as 'Squiz.Arrays.ArrayDeclaration.NoCommaAfterLast'. -->
+    <exclude name="SlevomatCodingStandard.Arrays.TrailingArrayComma"/>
+    <exclude name="SlevomatCodingStandard.Attributes.AttributeAndTargetSpacing"/>
+    <exclude name="SlevomatCodingStandard.Attributes.AttributesOrder"/>
+    <exclude name="SlevomatCodingStandard.Attributes.DisallowAttributesJoining"/>
+    <exclude name="SlevomatCodingStandard.Attributes.DisallowMultipleAttributesPerLine"/>
+    <exclude name="SlevomatCodingStandard.Attributes.RequireAttributeAfterDocComment"/>
+    <exclude name="SlevomatCodingStandard.Classes.BackedEnumTypeSpacing"/>
+    <exclude name="SlevomatCodingStandard.Classes.ClassLength"/>
+    <exclude name="SlevomatCodingStandard.Classes.ClassMemberSpacing"/>
+    <exclude name="SlevomatCodingStandard.Classes.ClassStructure"/>
+    <exclude name="SlevomatCodingStandard.Classes.ConstantSpacing"/>
+    <exclude name="SlevomatCodingStandard.Classes.DisallowConstructorPropertyPromotion"/>
+    <exclude name="SlevomatCodingStandard.Classes.DisallowMultiConstantDefinition"/>
+    <exclude name="SlevomatCodingStandard.Classes.DisallowMultiPropertyDefinition"/>
+    <exclude name="SlevomatCodingStandard.Classes.EmptyLinesAroundClassBraces"/>
```

Comment:
Proposing to enable it by default, because Drupal rules doesn't cover it, but with this configuration:

- `linesCountAfterOpeningBrace`: 1 - as it should be. Currently this is a valid one:
   ```php
   final class Foo {
   
   
      protected $foo = 'bar';
   ```
- `linesCountBeforeClosingBrace`: 1.

But there is a potential problem with tests. Such code will be invalid:

```php
    $element_a = new class() extends ElementBase {

    };

    $element_b = new class() extends ElementBase {

    };

    $element_a->addChild($element_b);
    $element_c = new class() extends ElementBase {

   };
```

It can be very frustrating, I think.